charts/rasa-x/templates/app-deployment.yaml

```diff
@@ -23,17 +23,7 @@ spec:
         {{- if .Values.app.podLabels }}
 {{ toYaml .Values.app.podLabels | indent 8 }}
         {{- end }}
-      {{- if .Values.app.podAnnotations }}
```

Comment:
Are these supposed to be removed?

Reply:
I don't know 😂 Did i miss some request related to removing this?

Reply:
Looks like my commit removed this, the missing pieces have been added back.